src/views/profile/index.vue

```diff
@@ -92,7 +92,7 @@ export default {
           callbacks: {
             click: () => {
               const next_url = this.$store.state.users.profile.is_face_code_set ? '/core/auth/profile/face/disable/' : '/core/auth/profile/face/enable/'
-              window.open(next_url, '_self')
+              window.open(next_url, '_blank')
             }
           }
         }
```

Comment:
In the provided code, there are no errors or discrepancies that I can see on these lines. The only adjustment could be minor formatting changes to ensure uniform coding style across similar sections of your project if you're working with multiple files.

However, since the question specifically asks for an English answer, it seems that I misunderstood your request. In actuality, this is an example showing how to use JavaScript to open a new tab using `window.open()` function.

The purpose and format of responses should remain consistent unless further clarifications about specific programming languages and frameworks are needed. Please let me know if anything else needs attention!